corehq/motech/repeaters/expression/repeater_generators.py

```diff
@@ -35,37 +36,25 @@ def get_url(self, repeat_record, url_template, payload_doc):
         }
         return url_template.format(
             **{
-                template_var: expressions[template_var](payload_doc) if template_var in expressions else ""
+                template_var: expressions[template_var](payload_doc_json) if template_var in expressions else ""
                 for template_var in required_template_vars
             }
         )
 
 
-class FormExpressionPayloadGenerator(ExpressionPayloadGenerator):
-    def get_payload(self, repeat_record, payload_doc, parsed_expression):
-        result = self._parse_payload(payload_doc, parsed_expression)
-        return json.dumps(result, cls=DjangoJSONEncoder)
-
-    def _parse_payload(self, payload_doc, parsed_expression):
-        payload_doc_json = payload_doc.to_json()
-        return parsed_expression(payload_doc_json, EvaluationContext(payload_doc_json))
-
-
-class ArcGISFormExpressionPayloadGenerator(FormExpressionPayloadGenerator):
+class ArcGISFormExpressionPayloadGenerator(ExpressionPayloadGenerator):
 
     def get_url(self, repeat_record, url_template, payload_doc):
-        if not (
-            toggles.UCR_EXPRESSION_REGISTRY.enabled(repeat_record.domain)
-            and toggles.ARCGIS_INTEGRATION.enabled(repeat_record.domain)
-        ):
+        if not toggles.ARCGIS_INTEGRATION.enabled(repeat_record.domain):
```

Comment:
The `UCR_EXPRESSION_REGISTRY` test is not necessary here since it is already happening in the super method